package.json

```diff
@@ -36,8 +36,8 @@
 		"docs:serve": "./node_modules/.bin/http-server docs/jsdoc/",
 		"precommit": "lint-staged && npm run lint",
 		"prepush": "npm run lint",
-		"snyk-protect": "snyk protect",
-		"prepublish": "npm run snyk-protect"
+		"check:dependencies": "./node_modules/.bin/snyk protect",
+		"prepublish": "npm run check:dependencies"
```

Comment:
From the npm docs:
> PREPUBLISH AND PREPARE
DEPRECATION NOTE
Since npm@1.1.71, the npm CLI has run the prepublish script for both npm publish and npm install, because it's a convenient way to prepare a package for use (some common use cases are described in the section below). It has also turned out to be, in practice, very confusing.

An alternative is to add an advisory `prepublishOnly` script, which will do nothing until we update our supported npm version to 4+, when it will run the script before publishing but not before installing.